src/main/java/by/dragonsurvivalteam/dragonsurvival/magic/abilities/SeaDragon/active/StormBreathAbility.java

```diff
@@ -367,6 +380,38 @@ public void sound(){
 
 	@Override
 	public void onBlock(BlockPos pos, BlockState blockState, Direction direction){
+		if (!(player.level instanceof ServerLevel serverLevel)) {
+			return;
+		}
+
+		if (blockState.getMaterial().isSolidBlocking()) {
+			if (/* 30% */ player.getRandom().nextInt(100) < 30) {
+				AreaEffectCloud entity = new AreaEffectCloud(EntityType.AREA_EFFECT_CLOUD, player.level);
+				entity.setWaitTime(0);
+				entity.setPos(pos.above().getX(), pos.above().getY(), pos.above().getZ());
+				entity.setPotion(new Potion(new MobEffectInstance(DragonEffects.CHARGED, /* Effect duration is normally divided by 4 */ Functions.secondsToTicks(10) * 4)));
+				entity.setDuration(Functions.secondsToTicks(2));
+				entity.setRadius(1);
+				entity.setParticle(new SmallLightningParticleData(37, false));
+				entity.setOwner(player);
+				serverLevel.addFreshEntity(entity);
+			}
+		}
+		
+		if (blockState.getMaterial().equals(Material.WATER)) {
+			if (/* 50% */player.getRandom().nextInt(100) < 30) {
```

Comment:
is the comment inaccurate (possible) or did sth. change here
would recommend changing either the comment or the value

Reply:
The comment is inaccurate, I changed the value a few times and ended up at 30%.  Even though there's a greater volume of blocks rolled each time due to affecting a volume instead of an area, the likelihood of each one triggering is balanced out by their reduced effect radius.  I've updated it.